syntax = "proto2";
package tutorial;


option go_package = "gateway/proto";

service Actuator{
    rpc ChangeState(ActuatorState) returns (ActuatorState) {}
}

message ActuatorState {
    optional string id = 1;
    optional string state = 2;
}

